testing/bdd/templates/timeBase_clock_clockMode_mandatory.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" ?>
+<tt:tt
+        ebuttp:sequenceIdentifier="testSequence001"
+        ebuttp:sequenceNumber="1"
+        ttp:timeBase="{{ time_base }}"
+{% if clock_mode %}
+  {% if clock_mode == "*?Empty?*" %}
+        ttp:clockMode=""
```

Comment:
When there is nothing in the `Examples` table, the `{% if clock_mode %}` is false and if you look at the template, then ttp:clockMode is never here. If we put `*?Empty?*` in the examples tables, then it is there and is `ttp:clockMode=""`. I can add an `*?Empty?*` test in the example table, then both cases will be covered.